Packs/Phishing/Playbooks/Phishing_-_Generic_v3_6_8.yml

```diff
@@ -3819,45 +3860,14 @@ inputs:
     For example: name of the organization finance app that the attacker might impersonate.
     This input is used in the "Spear Phishing Investigation" sub-playbook.
   playbookInputQuery:
-inputSections:
-- inputs:
-  - Role
-  - OnCall
-  - EmailHuntingCreateNewIncidents
-  - ListenerMailbox
-  - SendMailInstance
-  - UserEngagement
-  - TakeManualActions
-  name: Incident Management
-  description: Incident management settings and data, including escalation processes, and user engagements.
-- inputs:
-  - DetonateURL
-  - InternalRange
-  - InternalDomains
-  name: Enrichment
-  description: Enrichment settings and data, including assets and indicators enrichment using third-party enrichers.
-- inputs:
-  - AuthenticateEmail
-  - CheckMicrosoftHeaders
-  - GetOriginalEmail
-  - OriginalAuthenticationHeader
-  - PhishingModelName
-  - DBotPredictURLPhishingURLsNumber
-  - EmailFileToExtract
-  - HuntEmailIndicators
-  - KeyWordsToSearch
-  name: Investigation
-  description: Investigation settings and data, including any deep dive incident investigation and verdict determination.
-- inputs:
-  - BlockIndicators
-  - SearchAndDelete
-  - SearchAndDeleteIntegration
-  - O365DeleteType
-  - O365ExchangeLocation
-  - O365AllowNotFoundSearchLocations
-  - O365ExchangeLocationExclusion
-  name: Remediation
-  description: Remediation settings and data, including containment, eradication, and recovery.
+- key: NonMaliciousEmailReview
+  value:
+    simple: "False"
```

Comment:
please set it to true - so we won't make BC for old versions ( True means that the analyst still must review the incident right?)